index.html

```diff
@@ -1,298 +1,287 @@
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
               </div>
             </div>
           </div>
+        </div>
 
-          <div class="card">
-            <div class="data">
-              <div class="content">
-                <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
-                <div class="language">
-                  <ul>
-                    <li>JavaScript</li>
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Ruby on rails</li>
-                  </ul>
-                </div>
-                <div class="wrapper">
-                  <button type="button">See Project</button>
-                </div>
+        <div class="card">
+          <div class="data">
+            <div class="content">
+              <h3 class="project-title">Multiiii-Post-Stories Gain+Glory</h3>
+              <div class="language">
+                <ul>
+                  <li>JavaScript</li>
+                  <li>HTML</li>
+                  <li>CSS</li>
+                  <li>Ruby on rails</li>
+                </ul>
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
               </div>
             </div>
           </div>
+        </div>
 
-          <div class="card">
-            <div class="data">
-              <div class="content">
-                <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
-                <div class="language">
-                  <ul>
-                    <li>JavaScript</li>
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Ruby on rails</li>
-                  </ul>
-                </div>
-                <div class="wrapper">
-                  <button type="button">See Project</button>
-                </div>
+        <div class="card">
+          <div class="data">
+            <div class="content">
+              <h3 class="project-title">Multiiii-Post-Stories Gain+Glory</h3>
+              <div class="language">
+                <ul>
+                  <li>JavaScript</li>
+                  <li>HTML</li>
+                  <li>CSS</li>
+                  <li>Ruby on rails</li>
+                </ul>
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
               </div>
             </div>
           </div>
         </div>
       </div>
-    </section>
-    <!-- About Myself -->
-    <section class="about-me" id="about">
-      <div class="title">
-        <div class="img-content">
-          <h1>About me</h1>
-          <img src="./images/mask.svg" alt="" />
+    </div>
+  </section>
+  <!-- About Myself -->
+  <section class="about-me" id="about">
+    <div class="title">
+      <div class="img-content">
+        <h2>About me</h2>
+        <img src="./images/mask.svg" alt="" />
```

Comment:
- [ ] [OPTIONAL] Hi you have the `alt` attribute on all your `img` tags 👍 however, you would also need to specify the value of this attribute. this is important for accessibility because if the user cannot see the image they should at least be able to see a description of what the image represents

Reply:
Thank you very much for the feedback. However, this was also stated in our general comments(note) as part of the changes we intend to make.